.github/workflows/doc-generator.yml

```diff
@@ -0,0 +1,35 @@
+name: Comment on Failed Test
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  spec_doc_generator:
+    name: Check Generator Templates
+    runs-on: ubuntu-22.04
+    container:
+      image: crystallang/crystal
```

Comment:
Is it possible to pin this to a specific version?

Reply:
yes it is possible

Reply:
Please do